requirements.txt

```diff
@@ -20,3 +20,4 @@ umap-learn>=0.3.10
 legacy-api-wrap
 setuptools_scm
 packaging
+scikit-misc
```

Comment:
I'm not sure I want this to be a hard dependency. It doesn't look like it's been a particularly active repo.

I think I'm okay with it being an optional dependency since (looking through it's dependents) I see a fair amount of packages in our ecosystem.

Reply:
@ivirshup, `scikit-misc` is still actively maintained but it is mostly used through [plotnine](https://github.com/has2k1/plotnine), where it is optional dependency.